deployment/src/main/java/io/quarkiverse/qute/web/deployment/QuteWebTemplateBuildItem.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkiverse.qute.web.deployment;
+
+import static io.quarkiverse.qute.web.runtime.PathUtils.removeExtension;
+import static io.quarkiverse.qute.web.runtime.PathUtils.removeLeadingSlash;
+import static io.quarkiverse.qute.web.runtime.PathUtils.removeTrailingSlash;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class QuteWebTemplateBuildItem extends MultiBuildItem {
+
+    /**
+     * templatePath is used also as path if link is null (e.g. "my-blog-post")
+     */
+    private final String templatePath;
+
+    /**
+     * The link to use for this template or null to use the template path (e.g "posts/my-blog-post")
+     *
+     * If two links are identical, an exception is thrown
+     * If a link and a path are identical for different items, the link has priority
```

Comment:
We should have a test for these scenarios...

Reply:
ok adding them